hello_pull.txt

```diff
@@ -1 +1 @@
-Hello pull request tutorial
+This is important Hello pull request tutorial
```

Comment:
This is not necessary.

Reply:
Please provide more explanation

Reply:
In my opinion this can emphasize the importance of the Hello world!!

Reply:
LGTM
(Looks Good To Me)